manifests/m/Mirantis/Lens/5.1.0/Mirantis.Lens.yaml

```
@@ -0,0 +1,8 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Mirantis.Lens
+PackageVersion: 5.1.0
```

```suggestion
PackageVersion: 5.1.0-latest.20210715.1
```

Upgrade loop if the version doesn't match the one in Control Panel.

---

Thanks @ItzLevvie for the suggestion! I have already applied it. I will be looking at this for now on.